.pre-commit-config.yaml

```diff
@@ -0,0 +1,47 @@
+ci:
+  autofix_commit_msg: "chore(pre-commit): autofix run"
+  autoupdate_commit_msg: "chore(pre-commit): autoupdate hooks"
+
+default_install_hook_types:
+  - pre-commit
+  - commit-msg
+
+repos:
+  - repo: https://github.com/lmmx/just-pre-commit
+    rev: v0.6.5
+    hooks:
+      - id: run-just-rust
+        args: ['precommit']
+  - repo: https://github.com/compilerla/conventional-pre-commit
+    rev: v4.3.0
+    hooks:
+      - id: conventional-pre-commit
+        stages: [commit-msg]
+        # args: [build, chore, ci, docs, feat, fix, perf, refactor, revert, style, test]
```

Comment:
```suggestion
```
I don't want to enforce this to contributors, I prefer rewriting commit messages when merging using a squash commit.